codebuild/spec/buildspec_omnibus.yml

```diff
@@ -372,4 +372,3 @@ batch:
           LATEST_CLANG: true
           TESTS: fuzz
           FUZZ_TIMEOUT_SEC: 60
-
```

Comment:
Ignore this: Its showing up in git diff weirdly. I did not delete any line here - you can view the right diff in the latest commit.